base/src/com/thoughtworks/go/util/SystemEnvironment.java

```diff
@@ -184,6 +184,8 @@
     public static GoSystemProperty<Integer> GO_ENCRYPTION_API_MAX_REQUESTS = new GoIntSystemProperty("go.encryption.api.max.requests", 30);
 
     public static GoSystemProperty<Boolean> WEBSOCKET_ENABLED = new GoBooleanSystemProperty("go.agent.websocket.enabled", false);
+    public static GoSystemProperty<Boolean> CONSOLE_LOGS_THROUGH_WEBSOCKET_ENABLED = new GoBooleanSystemProperty("go.agent.console.logs.websocket.enabled", false);
```

Comment:
Does it make sense to set the default to `true` since this kicks in only when websockets is [turned on](https://github.com/kierarad/gocd/blob/2a909964c401ae088def31da815b56e7d6caf475/common/src/com/thoughtworks/go/work/DefaultGoPublisher.java#L82)? When we turn on websockets for any of the environments, console logging will start happening over websockets by default then. Given that we have this property available, we could always turn it off later if we find any issue with it.